### PR TITLE
Travis CI Config Sample 

### PR DESCRIPTION
**Pull request by TheDancerCodes**
This PR contains the Gradle Enterprise Build Config Samples for **Travis CI**.

It highlights the most common [Travis Environment Variables](https://docs.travis-ci.com/user/environment-variables/) that you would need as **CiMetadata** in your Build Scans.

### CODE REVIEW

**Comment by etiennestuder** [comment]
When we add this, we also need to do the same to the Maven script / Maven extension.

**Comment by TheDancerCodes** [comment]
> When we add this, we also need to do the same to the Maven script / Maven extension.

Hey @etiennestuder 
Pushing a maven one shortly.

**Comment by TheDancerCodes** [comment]
@etiennestuder 
I have implemented your feedback.

I believe the PR is good now.

File: common-custom-user-data-scripts/maven-common-custom-user-data.groovy [hunk 1]
```diff
@@ -254,6 +275,10 @@ static boolean isGitLab() {
     System.getenv('GITLAB_CI')
 }
 
+static boolean isTravis() {
+    System.getenv('GITLAB_CI')
```
Comment (runningcode) [inline common-custom-user-data-scripts/maven-common-custom-user-data.groovy:279]:
```suggestion
    System.getenv('TRAVIS_JOB_ID')
```

Comment (TheDancerCodes) [reply common-custom-user-data-scripts/maven-common-custom-user-data.groovy:279]: On it.

**Comment by TheDancerCodes** [comment]
@runningcode 
Feedback implemented.

**Comment by TheDancerCodes** [comment]
@runningcode 
All done.

**Comment by TheDancerCodes** [comment]
Hello @etiennestuder 
We have been running this in our GE setup for the last couple of weeks and I can confirm that it works as expected.

See the screenshots below.

---
**Travis CI Metadata:**

GE CI Tags

---

**Travis Custom Values:**

GE CI Tags 2

**Comment by runningcode** [comment]
thanks @TheDancerCodes !

**Comment by etiennestuder** [comment]
> When we add this, we also need to do the same to the Maven script / Maven extension.

@runningcode this got merged without ever having been applied to the Maven extension

**Comment by etiennestuder** [comment]
I'm confused. I see it on master but not in this PR. Seems it got done.

**Comment by runningcode** [comment]
Yup. I did it in a follow up: https://github.com/gradle/gradle-enterprise-build-config-samples/commit/126c9a44152f74abac9ce4f4bbffc577bc96bc77

**Pull request merged**